lib/components/artist/shows.js

```diff
@@ -0,0 +1,12 @@
+/* @flow */
+'use strict';
+
+import React from 'react-native';
+const { Text } = React;
+
+
+export default class Shows extends React.Component {
+	render() {
+		return <Text> Current & Upcoming Shows </Text>
+	}
```

Comment:
Also add trailing semi-colons. They are not required in JS, but for now let’s not make our lives harder by having to deal with those subtle syntax issues.